float32 x
float32 y
float32 w
int32 id 
---
string msg_feedback